inference-engine/tests/functional/plugin/cpu/single_layer_tests/pooling.cpp

```diff
@@ -4,77 +4,115 @@
 
 #include "ngraph_functions/builders.hpp"
 #include "test_utils/cpu_test_utils.hpp"
-#include "shared_test_classes/single_layer/pooling.hpp"
 #include "test_utils/fusing_test_utils.hpp"
+#include "shared_test_classes/single_layer/pooling.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
 
-using namespace InferenceEngine;
 using namespace CPUTestUtils;
-using namespace LayerTestsDefinitions;
 
 namespace CPULayerTestsDefinitions {
-typedef std::tuple<
-        poolLayerTestParamsSet,
-        CPUSpecificParams,
-        fusingSpecificParams
-> poolLayerCpuTestParamsSet;
+
+using PoolingInputShapes = std::pair<std::vector<ov::PartialShape>, std::vector<ov::Shape>>;
```

Comment:
You can use `ov::test::InputShape` type alias.

Reply:
But the idea is to use vector of `ov::PartialShape` per each input. Please see eltwise and softmax common test classes for the reference